metals/src/main/scala/scala/meta/internal/builds/BuildTools.scala

```diff
@@ -121,6 +122,26 @@ final class BuildTools(
   )
   def isBazel: Boolean = bazelProject.isDefined
 
+  private def customBsps: List[BspOnly] = {
+    val bspFolder = workspace.resolve(".bsp")
+    val root = customProjectRoot.getOrElse(workspace)
```

Comment:
Because we always create `.bsp` in the root I assumed we search only there. The `root` only says where the bsp command will be run. We can change this behaviour though.

Reply:
I think ideally, we would look for .bsp in the custom root also.